source/diagnostic.h

```diff
@@ -31,10 +31,23 @@
 #include <sstream>
 #include <utility>
 
-#include "spirv-tools/libspirv.h"
+#include <spirv-tools/libspirv.h>
+
+/// For debugging purposes only
+/// Prints the string to stdout
+#define MSG(msg)                                        \
+  do {                                                  \
+    libspirv::message(__FILE__, size_t(__LINE__), msg); \
+  } while (0)
+
+/// For debugging purposes only
+/// prints the variable value/location/and name to stdout
+#define SHOW(exp)                                               \
+  do {                                                          \
+    libspirv::message(__FILE__, size_t(__LINE__), #exp, (exp)); \
```

Comment:
Does a use of this macro even compile?  The message method only takes 3 parameters.